src/views/List.jsx

```diff
@@ -33,6 +33,7 @@ export function List({ data, listToken }) {
 
 	return data.length > 0 ? (
 		<>
+			<p>{listToken}</p>
```

Comment:
We added this line so that we could see the list token on the page. We figured it will continue to be helpful.

Reply:
I love this! I wonder if we should should always be showing it even when the list is empty. Also, maybe we could have have it be something like `S <p>Shopping list token: {listToken}</p>` so that users know what it is?

Reply:
Oh that's great! Actually we added it for developers but yes, I agree, users would probably like it as well :)